### ISSUE TITLE
Mailing list (sign up to receive RFCs)

### ISSUE THREAD

**Issue by hoodmane** [pyodide/pyodide#1546]
In the future, we will send out requests for comment to the Pyodide mailing list instead of @ ing people here. If you are interested, please sign up here:
https://mail.python.org/mailman3/lists/pyodide.python.org/

@albertas @casatir @ChristopherHButler @daoxian @dmondev @gzuidhof @JackChen20200818 @jmsmdy @joemarshall @johnwason @josephrocca @jps1 @leopsidom @oeway @ondrej-stanek-ozobot @therealadityashankar

**Comment by rth** [comment]
The idea is that the detailed technical discussion will still happen on Github, but because it's hard to follow all the notifications we will also post most important request for comments or announcements on the mailing list. You are also of course welcome to use that mailing list for general discussions.

See https://github.com/pyodide/pyodide#communication for all the current communications channels.

**Comment by rth** [comment]
Closing since the mailing list has been created a while ago and interested people have subscribed.
